### PR TITLE
Update releasenotes.md

### PR DESCRIPTION
**Pull request by jasonstephen15**
Updating release notes for 0.9.622 and 0.9.627 - prerelease

### CODE REVIEW

File: microsoft-edge/webview2/releasenotes.md [hunk 1]
```diff
@@ -20,6 +20,46 @@ The WebView2 team is delivering updates to the [WebView2 SDK][NuGetGallery] on a
...
+    > **Announcement**:  Visual Hosting is now in preview.
+
+*   Disconnected [browser policies](https://docs.microsoft.com/en-us/deployedge/microsoft-edge-policies). 
+*   Added [IsSingleSignOnUsingOSPrimaryAccountEnabled][ReferenceWin3209538Icorewebview2experimentaloptionsGetIssinglesignonusingosprimaryaccountenabled] property on WebView2 environment options to enable conditional access for WebView.
+*   Added 
+*   Fixed System.Windows.Rect.  \([\#235][GithubMicrosoftedgeWebviewfeedbackIssue235]\).
+*   Updated the Windows.open() function to handle "no parameter" case.  \([\#293][GithubMicrosoftedgeWebviewfeedbackIssue293]\).
+
+
+#### .NET
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/releasenotes.md:58]: These .NET items should be under 0.9.627-prerelease. There will be no .NET bits in 0.9.622.

File: microsoft-edge/webview2/releasenotes.md [hunk 2]
```diff
@@ -20,6 +20,46 @@ The WebView2 team is delivering updates to the [WebView2 SDK][NuGetGallery] on a
...
+
+#### General  
+
+*   > [!IMPORTANT]
+    > **Announcement**: This SDK release is a candidate for WebView2 Win32 C++ GA. 
+* > [!IMPORTANT]
+    > **Announcement**:  Visual Hosting is now in preview.
+
+*   Disconnected [browser policies](https://docs.microsoft.com/en-us/deployedge/microsoft-edge-policies). 
+*   Added [IsSingleSignOnUsingOSPrimaryAccountEnabled][ReferenceWin3209538Icorewebview2experimentaloptionsGetIssinglesignonusingosprimaryaccountenabled] property on WebView2 environment options to enable conditional access for WebView.
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/releasenotes.md:52]: The API is renamed to `AllowSingleSignOnUsingOSPrimaryAccount` and link to the doc is TBD.

File: microsoft-edge/webview2/releasenotes.md [hunk 3]
```diff
@@ -20,6 +20,46 @@ The WebView2 team is delivering updates to the [WebView2 SDK][NuGetGallery] on a
...
+## 0.9.622  
+Release Date: 09/08/2020
+
+[NuGet package][NuGetGallery0.9.579] \| minimum Microsoft Edge version 86.0.579.0.  
+
+#### General  
+
+*   > [!IMPORTANT]
+    > **Announcement**: This SDK release is a candidate for WebView2 Win32 C++ GA. 
+* > [!IMPORTANT]
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/releasenotes.md:48]:
```suggestion
*   > [!IMPORTANT]
```

File: microsoft-edge/webview2/releasenotes.md [hunk 4]
```diff
@@ -20,6 +20,46 @@ The WebView2 team is delivering updates to the [WebView2 SDK][NuGetGallery] on a
...
+
+#### General  
+* 
+#### .NET  
+
+*   .NET Binaries are now [Strongly Named]().  \([\#181][GithubMicrosoftedgeWebviewfeedbackIssue181]\).
+
+
+## 0.9.622  
+Release Date: 09/08/2020
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/releasenotes.md:40]: Release date would most likely be next Wednesday. Adding a comment here to ensure we don't forget to update it :)

File: microsoft-edge/webview2/releasenotes.md [hunk 5]
```diff
@@ -20,6 +20,46 @@ The WebView2 team is delivering updates to the [WebView2 SDK][NuGetGallery] on a
...
+
+[NuGet package][NuGetGallery0.9.579] \| minimum Microsoft Edge version 86.0.579.0.
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/releasenotes.md:42]: Check with @pagoe-msft to see whether we should use WebView2 Runtime version for release packages and Canary version for prerelease packages.

**Review by peiche-jessica — CHANGES_REQUESTED** [review]

File: microsoft-edge/webview2/releasenotes.md (hunk 5 above)
Comment (jasonstephen15) [reply microsoft-edge/webview2/releasenotes.md:42]: @pagoe-msft Do you have insight into Jessica's comment above

File: microsoft-edge/webview2/releasenotes.md [hunk 6]
```diff
@@ -12,29 +12,66 @@ keywords:  IWebView2, IWebView2WebView, webview2, webview, win32 apps, win32, ed
...
+Release Date: Sept 09, 2020
+
+[NuGet package][NuGetGallery0.9.579] \| minimum Microsoft Edge version 86.0.579.0.  
+
+#### General  
+
+*   > [!IMPORTANT]
+    > **Announcement**: This SDK release is a candidate for WebView2 Win32 C++ GA. 
+*   > [!IMPORTANT]
+    > **Announcement**:  Visual Hosting is now in preview.
```
Comment (liminzhu) [inline microsoft-edge/webview2/releasenotes.md:51]: Visual hosting in preview in Pre-release. You probably want to also briefly explain what visual hosting is and give some pointers. You can work with either Brad or me on that.

File: microsoft-edge/webview2/releasenotes.md [hunk 7]
```diff
@@ -12,29 +12,66 @@ keywords:  IWebView2, IWebView2WebView, webview2, webview, win32 apps, win32, ed
...
+
+## 0.9.622  
+Release Date: Sept 09, 2020
+
+[NuGet package][NuGetGallery0.9.579] \| minimum Microsoft Edge version 86.0.579.0.  
+
+#### General  
+
+*   > [!IMPORTANT]
+    > **Announcement**: This SDK release is a candidate for WebView2 Win32 C++ GA.
```
Comment (liminzhu) [inline microsoft-edge/webview2/releasenotes.md:49]: I would say "SDK 0.9.622 is the release candidate for WebView2 Win32 C/C++ GA. The GA version will likely have the same API interface and functionality."

File: microsoft-edge/webview2/releasenotes.md [hunk 8]
```diff
@@ -12,29 +12,68 @@ keywords:  IWebView2, IWebView2WebView, webview2, webview, win32 apps, win32, ed
...
 > Re-compile your app after updating the NuGet package.  
 
 > [!IMPORTANT]
 > While WebView2 is a preview, the .NET APIs are in the `pre-release package`.  
 
+
+## 0.9.628-prerelease  
+Release Date: Sept 09, 2020
+
+[NuGet package][NuGetGallery0.9.579] \| minimum Microsoft Edge version 86.0.579.0.
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/releasenotes.md:27]: Please update release date as it is postponed until today, add two nuget package entries, and confirm the minimum supported version.

File: microsoft-edge/webview2/releasenotes.md [hunk 9]
```diff
@@ -12,29 +12,68 @@ keywords:  IWebView2, IWebView2WebView, webview2, webview, win32 apps, win32, ed
...
+    > **Announcement**: This prerelease will continue to release out of Edge 87 Branch. In the future, pre-release SDK releases will match the Edge Canary builds, whereas our normal releases will be in sync with Edge Stable.
+
+#### General
+*   > [!IMPORTANT]
+    > **Announcement**:  Visual Hosting is now in preview.
+* Added the CompositionController as an experimental interface. This lets your app connect WebView2 to Dcomp visual or WinComp ContainerVisual and provides API that allows the app to forward mouse and touch inputs to the WebView.
+
+#### .NET  
+
+*   .NET Binaries are now [Strongly Named]().  \([\#181][GithubMicrosoftedgeWebviewfeedbackIssue181]\).
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/releasenotes.md:39]:
```suggestion
*   .NET Binaries are now [strongly-named](https://docs.microsoft.com/dotnet/standard/assembly/strong-named).  \([\#181][GithubMicrosoftedgeWebviewfeedbackIssue181]\).
```

File: microsoft-edge/webview2/releasenotes.md [hunk 10]
```diff
@@ -12,29 +12,68 @@ keywords:  IWebView2, IWebView2WebView, webview2, webview, win32 apps, win32, ed
...
+*   Updated NuGet Target to include WebViewLoader.dll.  \([\#228][GithubMicrosoftedgeWebviewfeedbackIssue228]\) & \([\#183][GithubMicrosoftedgeWebviewfeedbackIssue183]\).
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/releasenotes.md:40]:
```suggestion
*   Updated NuGet Target to include WebView2Loader.dll.  \([\#228][GithubMicrosoftedgeWebviewfeedbackIssue228]\) & \([\#183][GithubMicrosoftedgeWebviewfeedbackIssue183]\).
```

File: microsoft-edge/webview2/releasenotes.md [hunk 11]
```diff
@@ -12,29 +12,68 @@ keywords:  IWebView2, IWebView2WebView, webview2, webview, win32 apps, win32, ed
...
+*   Updated WebResourceRequested to expose HttpRequestMessage and HttpResponseMessage APIs in .NET.  \([\#131][GithubMicrosoftedgeWebviewfeedbackIssue131]\).
+
+
+## 0.9.622  
+Release Date: Sept 09, 2020
+
+[NuGet package][NuGetGallery0.9.579] \| minimum Microsoft Edge version 86.0.579.0.
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/releasenotes.md:47]: Same as above.

File: microsoft-edge/webview2/releasenotes.md [hunk 12]
```diff
@@ -12,29 +12,68 @@ keywords:  IWebView2, IWebView2WebView, webview2, webview, win32 apps, win32, ed
...
+
+[NuGet package][NuGetGallery0.9.579] \| minimum Microsoft Edge version 86.0.579.0.  
+
+#### General  
+
+*   > [!IMPORTANT]
+    > **Announcement**: This SDK is the Release Candidate for WebView2 Win32 C/C++ GA. The GA version will likely have the same API interface and functionality.
+
+
+*   Disconnected [browser policies](https://docs.microsoft.com/en-us/deployedge/microsoft-edge-policies).
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/releasenotes.md:55]: Avoid hard-coded locale code in the links. The warnings can be found in the Docs Build Status.
```suggestion
*   Disconnected [browser policies](https://docs.microsoft.com/deployedge/microsoft-edge-policies). 
```

File: microsoft-edge/webview2/releasenotes.md [hunk 13]
```diff
@@ -12,29 +12,68 @@ keywords:  IWebView2, IWebView2WebView, webview2, webview, win32 apps, win32, ed
...
+[NuGet package][NuGetGallery0.9.579] \| minimum Microsoft Edge version 86.0.579.0.  
+
+#### General  
+
+*   > [!IMPORTANT]
+    > **Announcement**: This SDK is the Release Candidate for WebView2 Win32 C/C++ GA. The GA version will likely have the same API interface and functionality.
+
+
+*   Disconnected [browser policies](https://docs.microsoft.com/en-us/deployedge/microsoft-edge-policies). 
+*   Added [AllowSingleSignOnUsingOSPrimaryAccount][ReferenceWin3209538Icorewebview2experimentaloptionsGetIssinglesignonusingosprimaryaccountenabled] property on WebView2 environment options to enable conditional access for WebView.
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/releasenotes.md:56]: The path for AllowSingleSignOnUsingOSPrimaryAccount would be `./reference/win32/0-9-622/icorewebview2environmentoptions.md#get_allowsinglesignonusingosprimaryaccount`

File: microsoft-edge/webview2/releasenotes.md [hunk 14]
```diff
@@ -12,29 +12,68 @@ keywords:  IWebView2, IWebView2WebView, webview2, webview, win32 apps, win32, ed
...
+*   Updated ICoreWebView2NewWindowRequestedEventArgs to include WindowFeatures property, and its associated ICoreWebView2WindowFeatures.  \([\#293][GithubMicrosoftedgeWebviewfeedbackIssue293]\).
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/releasenotes.md:57]: Link ICoreWebView2NewWindowRequestedEventArgs  to `./reference/win32/0-9-622/icorewebview2newwindowrequestedeventargs.md`.

File: microsoft-edge/webview2/releasenotes.md [hunk 15]
```diff
@@ -12,29 +12,68 @@ keywords:  IWebView2, IWebView2WebView, webview2, webview, win32 apps, win32, ed
...
+*   Updated System.Windows.Rect. to use System.Drawing.Rectangle instead of System.Windows.Rect \([\#235][GithubMicrosoftedgeWebviewfeedbackIssue235]\).
+*   Updated the Windows.open() function to handle "no parameter" case.  \([\#293][GithubMicrosoftedgeWebviewfeedbackIssue293]\).
+*   [AdditionalBrowserArguments](https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/0-9-538/icorewebview2environmentoptions#put_additionalbrowserarguments) set via ICoreWebView2EnvironmentOptions will not be overridden with environment variable nor registry value. See [CreateCoreWebView2EnvironmentWithOptions](https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/0-9-538/webview2-idl#createcorewebview2environmentwithoptions) for more details.
+
 ## 0.9.579  
-Release Date: 07/20/2020
+Release Date: July 20, 2020
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/releasenotes.md:63]: Shall we change all date formats?

File: microsoft-edge/webview2/releasenotes.md [hunk 16]
```diff
@@ -12,29 +12,68 @@ keywords:  IWebView2, IWebView2WebView, webview2, webview, win32 apps, win32, ed
...
+*   Updated ICoreWebView2NewWindowRequestedEventArgs to include WindowFeatures property, and its associated ICoreWebView2WindowFeatures.  \([\#293][GithubMicrosoftedgeWebviewfeedbackIssue293]\).
+*   Updated System.Windows.Rect. to use System.Drawing.Rectangle instead of System.Windows.Rect \([\#235][GithubMicrosoftedgeWebviewfeedbackIssue235]\).
+*   Updated the Windows.open() function to handle "no parameter" case.  \([\#293][GithubMicrosoftedgeWebviewfeedbackIssue293]\).
+*   [AdditionalBrowserArguments](https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/0-9-538/icorewebview2environmentoptions#put_additionalbrowserarguments) set via ICoreWebView2EnvironmentOptions will not be overridden with environment variable nor registry value. See [CreateCoreWebView2EnvironmentWithOptions](https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/0-9-538/webview2-idl#createcorewebview2environmentwithoptions) for more details.
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/releasenotes.md:60]:
```suggestion
*   [AdditionalBrowserArguments](./reference/win32/0-9-622/icorewebview2environmentoptions#put_additionalbrowserarguments) set via ICoreWebView2EnvironmentOptions will not be overridden with environment variable nor registry value. See [CreateCoreWebView2EnvironmentWithOptions](./reference/win32/0-9-622/webview2-idl#createcorewebview2environmentwithoptions) for more details.
```

File: microsoft-edge/webview2/releasenotes.md [hunk 17]
```diff
@@ -12,29 +12,68 @@ keywords:  IWebView2, IWebView2WebView, webview2, webview, win32 apps, win32, ed
...
+
+*   > [!IMPORTANT]
+    > **Announcement**: This SDK is the Release Candidate for WebView2 Win32 C/C++ GA. The GA version will likely have the same API interface and functionality.
+
+
+*   Disconnected [browser policies](https://docs.microsoft.com/en-us/deployedge/microsoft-edge-policies). 
+*   Added [AllowSingleSignOnUsingOSPrimaryAccount][ReferenceWin3209538Icorewebview2experimentaloptionsGetIssinglesignonusingosprimaryaccountenabled] property on WebView2 environment options to enable conditional access for WebView.
+*   Updated ICoreWebView2NewWindowRequestedEventArgs to include WindowFeatures property, and its associated ICoreWebView2WindowFeatures.  \([\#293][GithubMicrosoftedgeWebviewfeedbackIssue293]\).
+*   Updated System.Windows.Rect. to use System.Drawing.Rectangle instead of System.Windows.Rect \([\#235][GithubMicrosoftedgeWebviewfeedbackIssue235]\).
+*   Updated the Windows.open() function to handle "no parameter" case.  \([\#293][GithubMicrosoftedgeWebviewfeedbackIssue293]\).
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/releasenotes.md:59]: "Updated NewWindowRequested event to handle window.open() call without parameters"?

File: microsoft-edge/webview2/releasenotes.md [hunk 18]
```diff
@@ -12,29 +12,68 @@ keywords:  IWebView2, IWebView2WebView, webview2, webview, win32 apps, win32, ed
...
+
+[NuGet package][NuGetGallery0.9.579] \| minimum Microsoft Edge version 86.0.579.0.  
+
+   > [!IMPORTANT]
+    > **Announcement**: This prerelease will continue to release out of Edge 87 Branch. In the future, pre-release SDK releases will match the Edge Canary builds, whereas our normal releases will be in sync with Edge Stable.
+
+#### General
+*   > [!IMPORTANT]
+    > **Announcement**:  Visual Hosting is now in preview.
+* Added the CompositionController as an experimental interface. This lets your app connect WebView2 to Dcomp visual or WinComp ContainerVisual and provides API that allows the app to forward mouse and touch inputs to the WebView.
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/releasenotes.md:35]:
```suggestion
* Added the CompositionController as an experimental interface. This lets your app connect WebView2 to DComp visual or WinComp ContainerVisual and provides API that allows the app to forward mouse and touch inputs to the WebView.
```

File: microsoft-edge/webview2/releasenotes.md [hunk 19]
```diff
@@ -12,37 +12,77 @@ keywords:  IWebView2, IWebView2WebView, webview2, webview, win32 apps, win32, ed
...
+
+#### General  
+
+*   > [!IMPORTANT]
+    > **Announcement**:  Visual Hosting is now in preview.  
+*   Added the CompositionController as an experimental interface.  This lets your app connect WebView2 to Dcomp visual or WinComp ContainerVisual and provides API that allows the app to forward mouse and touch inputs to the WebView.  
+
+#### .NET  
+
+*   .NET Binaries are now [Strongly Named]().  \([\#181][GithubMicrosoftedgeWebviewfeedbackIssue181]\).
```
Comment (jm-trd-ms) [inline microsoft-edge/webview2/releasenotes.md:40]: Please provide a link to the `Strongly Named` article.

**Review by peiche-jessica — APPROVED** [review]

**Review by jm-trd-ms — APPROVED** [review]

File: microsoft-edge/webview2/concepts/versioning.md [hunk 20]
```diff
@@ -12,34 +12,45 @@ keywords: IWebView2, IWebView2WebView, webview2, webview, wpf apps, wpf, edge, I
...
 In preview, the release package contains the following.  
 
 *   [Win32 C/C++ APIs][ReferenceWin3209622]: APIs in the SDK that are expected to remain the same at GA.  
 
-In preview, the pre-release package contains the following components.  
+In preview, the prerelease package contains the following components.  
 
-*   .NET APIs: [WPF][ReferenceWpf09515], [WinForms][ReferenceWinforms09515], and [Core][ReferenceDotnet09628]  
-*   Experimental APIs.  For more information, see the [Experimental APIs](#experimental-apis) section.  
+*   .NET APIs: [WPF][ReferenceWpf09515], [WinForms][ReferenceWinforms09515], and [Core][ReferenceDotnet09538]
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/concepts/versioning.md:30]: Can we change this back to the latest 628 please?

Comment (jm-trd-ms) [reply microsoft-edge/webview2/concepts/versioning.md]: Updated.

**Review by pagoe-msft — APPROVED** [review]

**Review by peiche-jessica — APPROVED** [review]
#sign-off
Thanks @jm-trd-ms

**Comment by jm-trd-ms** [comment]
#sign-off and merged.

**Pull request merged**